open_src/influx/meta/data.go

```diff
@@ -1760,14 +1760,14 @@ func (data *Data) pruneShardGroups(id uint64) error {
 	return nil
 }
 
-// validateURL returns an error if the URL does not have a port or uses a scheme other than UDP or HTTP.
+// validateURL returns an error if the URL does not have a port or uses a scheme other than HTTP.
 func validateURL(input string) error {
 	u, err := url.Parse(input)
 	if err != nil {
 		return ErrInvalidSubscriptionURL(input)
 	}
 
-	if u.Scheme != "udp" && u.Scheme != "http" && u.Scheme != "https" {
+	if u.Scheme != "http" && u.Scheme != "https" {
```

Comment:
Why not support `udp`?

Reply:
I don't know if openGemini support `UDP` write or not

Reply:
Maybe openGemini should support `UDP` protocol in the near future.